nvme-builtin.h

```diff
@@ -105,7 +105,7 @@ COMMAND_LIST(
 	ENTRY("virt-mgmt", "Manage Flexible Resources between Primary and Secondary Controller ", virtual_mgmt)
 	ENTRY("rpmb", "Replay Protection Memory Block commands", rpmb_cmd)
 	ENTRY("lockdown", "Submit a Lockdown command,return result", lockdown_cmd)
-	ENTRY("dim", "Send Discovery Information Management command to a Discovery Controller", dim_cmd) \
+	ENTRY("dim", "Send Discovery Information Management command to a Discovery Controller", dim_cmd)
```

Comment:
unrelated change

Reply:
checkpatch.pl did this....

Reply:
I just ran checkpatch again and it removed the second continueation as well.

Reply:
checkpatch.pl was complaining about these extra `\` so I removed them.